docs/codeql/codeql-language-guides/analyzing-data-flow-in-go.rst

```diff
@@ -188,7 +188,7 @@ These predicates are defined in the configuration:
 
 -  ``isSource`` - defines where data may flow from.
 -  ``isSink`` - defines where data may flow to.
--  ``isBarrier`` - optional, restricts the data flow.
+-  ``isBarrier`` - optional, defines where data flow is blocked.
```

Comment:
Minor: There's probably a better way to word this still, especially to better communicate that this is a desirable thing to do, but this shouldn't _block_ this PR.